test/ci/buildkite/test_runner.py

```diff
@@ -86,13 +91,31 @@ def main():
 
     # Decide which tests to run
     if (arguments.test_cpp):
-        run_ngtf_cpp_gtests(arguments.artifacts_dir, './', None)
+        test_filter = None
+        if arguments.backend:
+            if 'GPU' in arguments.backend:
+                test_filter = str(
+                    "-ArrayOps.Quanti*:ArrayOps.Dequant*:BackendManager.BackendAssignment:"
+                    "MathOps.AnyKeepDims:MathOps.AnyNegativeAxis:MathOps.AnyPositiveAxis:"
+                    "MathOps.AllKeepDims:MathOps.AllNegativeAxis:MathOps.AllPositiveAxis:"
+                    "NNOps.Qu*:NNOps.SoftmaxZeroDimTest*:"
+                    "NNOps.SparseSoftmaxCrossEntropyWithLogits")
```

Comment:
Minor:
lets add a `else <fail>` here, since only GPU backend is in the test setup right now